src/test/java/HomeWork4/tests/ProductTests.java

```diff
@@ -11,5 +11,6 @@ public void addToCartProductTest ()  {
         Assert.assertEquals(productsPage.getProductPrice(productName), "$7.99");
         Assert.assertEquals(productsPage.getProductDescription(productName), "Rib snap infant onesie for the junior automation engineer in development. Reinforced 3-snap bottom closure, two-needle hemmed sleeved and bottom won't unravel.");
         productsPage.clickAddToCartButton(productName);
+        Assert.assertEquals(productsPage.getNameOfButtonRemoveProduct(productName), "Remove", "Button is not select");
```

Comment:
Seriously, You check the text of the button? - that's very bad assertion. 
If want to check that item is added to the cart - you should go to the cart and verify it's present there